pipelines/cohort_screening/src/variants_report.py

```diff
@@ -0,0 +1,433 @@
+                    if ('benign' in clinvar_dict['significance']
+                            and clinvar_dict['reviewStatus'] not in review_status_disregard):
+                        logging.info(f'benign in significance, do not disregard, removing variant {clinvar_dict}')
+                        variant_dict[clinvar_field].remove(clinvar_dict)
+                    elif 'significance' in clinvar_dict and ('pathogenic' not in clinvar_dict['significance']
+                                                             and 'likely pathogenic'
+                                                             not in clinvar_dict['significance']) \
+                            and clinvar_dict['reviewStatus'] not in review_status_disregard:
+                        logging.info(f'ClinVar significance does not pass inclusion filters, /'
+                                     f'removing variant' + str(variant_dict))
+                        variant_dict[clinvar_field].remove(clinvar_dict)
+
+    # remove any variants that don't have any populated clinvar fields left
+    for position in filtered_positions[:]:
+        for variant_dict in position[variants_field][:]:
+            if len(variant_dict[clinvar_field]) == 0:
+                logging.info(f'no ClinVar fields remaining for variant, /'
+                             f'removing variant' + str(variant_dict))
+                position[variants_field].remove(variant_dict)
+
+    # remove any positions that don't have any variants left
+    for position in filtered_positions[:]:
+        if variants_field in position and len(position[variants_field]) == 0:
+            logging.info(f'no variants found for position, removing position' + str(position))
+            filtered_positions.remove(position)
+            removed_positions += 1
+
+    # Criteria 2 & 3
+    # remove any variants where genotype is 0/0, ./N, N/., or ./.
+    print('Filtering variants for inclusion in report based on genotype...')
+    pattern = r"\./|./\."
+    for position in filtered_positions[:]:
+        for samples_dict in position['samples'][:]:
+            # if any of the genotypes match the pattern, remove the variant
+            if re.match(pattern, samples_dict['genotype']) or samples_dict['genotype'] == '0/0':
+                logging.info(f'genotype does not pass inclusion filters, removing sample' + str(samples_dict))
+                position['samples'].remove(samples_dict)
+
+    # remove any positions that don't have any samples left
+    for position in filtered_positions[:]:
+        if 'samples' in position and len(position['samples']) == 0:
+            logging.info(f'position has no samples left, removing position' + str(position))
+            filtered_positions.remove(position)
+            removed_positions += 1
+
+    # # for testing
+    # with open("filtered_positions_criteria_1.json", 'w') as outfile:
+    #     json.dump(filtered_positions, outfile, indent=4)
```

Comment:
Maybe we can move this to _test_variants_report.py_

Reply:
I just removed it - if we get to a point where we need to do more heavy dev again we can add it back locally